src/tests/system/tests/test_failover.py

```diff
@@ -0,0 +1,45 @@
+"""
+SSSD Failover tests.
+
+:requirement: IDM-SSSD-REQ : LDAP Provider
```

Comment:
@danlavu Hi Dan, what should I use here?

Reply:
@jakub-vavra-cz

Reply:
:requirement: Failover